src/content/1.7/modules/sample_modules/order_view_page_new_hooks/additional-action-buttons.md

```diff
@@ -8,3 +8,61 @@ weight: 3
 ## `actionGetAdminOrderButtons` hook demo
 
 We use this hook to display additional action buttons into the main buttons bar.
+
+Let's add hook related code to main module class `demovieworderhooks`:
+
+```php
+    /**
+     * Add buttons to main buttons bar
+     */
+    public function hookActionGetAdminOrderButtons(array $params)
+    {
+        $order = new Order($params['id_order']);
+
+        /** @var \Symfony\Bundle\FrameworkBundle\Routing\Router $router */
+        $router = $this->get('router');
+
+        /** @var \PrestaShopBundle\Controller\Admin\Sell\Order\ActionsBarButtonsCollection $bar */
```

Comment:
I understand the use of fully qualified class names, especially in this context of partial code
But we could use the tutorial as an occasion to teach the reader how to use the `use` statements, in a note block below this code maybe?

Reply:
hm, not sure how to teach them, usually PHPSTORM helps doing that but not everybody uses it

Reply:
that's one of the good practices we try to force on ourselves in the core code (especially for built in objects like `Exception` ) But I guess in this case it's anecdotic, let's not focus on this detail